pkg/monitor/datapath_drop.go

```diff
@@ -43,30 +45,43 @@ type DropNotify struct {
 }
 
 // DumpInfo prints a summary of the drop messages.
-func (n *DropNotify) DumpInfo(data []byte) {
-	fmt.Printf("xx drop (%s) flow %#x to endpoint %d, identity %s->%s: %s\n",
-		api.DropReason(n.SubType), n.Hash, n.DstID, n.SrcLabel, n.DstLabel,
-		GetConnectionSummary(data[DropNotifyLen:]))
+func (n *DropNotify) DumpInfo(data []byte, numeric bool) {
+	buf := bufio.NewWriter(os.Stdout)
+	fmt.Fprintf(buf, "xx drop (%s) flow %#x to endpoint %d",
+		api.DropReason(n.SubType), n.Hash, n.DstID)
+	if numeric {
+		fmt.Fprintf(buf, ", identity %s->%d", n.SrcLabel, n.DstLabel)
```

Comment:
Was this supposed to be `%d->%d` instead of `%s->%d`?

(A helper function for the 4 lines to print the identity might help here since we seem to be using it a couple of times)